src/Microsoft.TestPlatform.Common/Resources/Resources.Designer.cs

```diff
@@ -349,7 +349,7 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Unable to find a datacollector with friendly name &apos;[0}&apos;..
+        ///   Looks up a localized string similar to Unable to find a datacollector with friendly name &apos;{0}&apos;..
```

Comment:
this fix seem generate resources file on build should I upload?

![image](https://user-images.githubusercontent.com/7894084/67486390-3d7cca00-f66c-11e9-9050-e5f81ca3288d.png)

Reply:
done